ssz-rs/benches/compute_proof.rs

```diff
@@ -0,0 +1,56 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use std::hint::black_box;
+
+use ssz_rs::{List, PathElement, Prove};
+use std::convert::TryFrom;
+
+/// Generate fake transactions to benchmark with.
+fn generate_dummy_transactions(size: usize) -> List<[u8; 32], 1000> {
+    let txs = (0..size).map(|i| [i as u8; 32]).collect::<Vec<_>>();
+    List::try_from(txs).expect("Failed to create transaction list")
+}
```

Comment:
I think using fake hashes is not a sufficiently complex input -- for instance the chunking process would be trivial because the data is already the size of a chunk.
It'd be better to generate fake transactions by taking some transfers and adding random calldata.

Reply:
Yeah or if we want to generate random data just generate a vec of bytes per transaction with length between `0` and `MAX_BYTES_PER_TRANSACTION` (and use random seeds so we can reproduce)

Reply:
How normal is transactions of size `1073741824` (`MAX_BYTES_PER_TRANSACTION`)?
Generating random data close to that size will significantly slow down the benchmark. Are there any statistics on distribution of tx sizes that I can use instead of just random data?